CHANGES.md

```diff
@@ -83,6 +83,11 @@ OpenSSL 3.0
 
    *Matt Caswell*
 
+ * Parallel dual-prime 1536/2048-bit modular exponentiation for
+   AVX512_IFMA capable processors.
+
+   *Ilya Albrekht, Sergey Kirillov, Andrey Matyukov (Intel Corp)*
```

Comment:
We'll need an ICLA from Ilya Albrekht and and ICLA and a CCLA update for Sergey Kirillov.

Reply:
Working on this now.

Reply:
The last commit should fix CLA issues.